plugins/fusion/qt.py

```diff
@@ -163,22 +163,28 @@ def on_new_window(self, window):
         # Called on initial plugin load (if enabled) and every new window; only once per window.
         wallet = window.wallet
 
-        if not (can_fuse_from(wallet) and can_fuse_to(wallet)):
-            # don't do anything with non-fusable wallets
-            # (if inter-wallet fusing is added, this should change.)
-            return
-
-        want_autofuse = Conf(wallet).autofuse
-        self.add_wallet(wallet, window.gui_object.get_cached_password(wallet))
+        can_fuse = can_fuse_from(wallet) and can_fuse_to(wallet)
+        if can_fuse:
+            sbbtn = FusionButton(self, wallet)
+            self.server_status_changed_signal.connect(sbbtn.update_server_error)
+        else:
+            # If we can not fuse we create a dummy fusion button that just displays a message
+            sbbtn = DisabledFusionButton(wallet, _('disabled because this wallet type is not supported'))
```

Comment:
On macOS the title doesn't always get shown in windows (stupid macos).  Also I wonder if the message can be made a little more "informative" and "helpful".. 

Something like...

`_('This wallet type ({wtype}) is cannot be used with CashFusion.\n\nPlease use a standard deterministic spending wallet with CashFusion.').format(wtype=wallet.wallet_type`

?

Reply:
updated